internal/api/ui/login/external_provider_handler.go

```diff
@@ -533,8 +533,8 @@ func (l *Login) externalUserNotExisting(w http.ResponseWriter, r *http.Request,
 		}
 	}
 
-	// if auto creation or creation itself is disabled, send the user to the notFoundOption
-	if !provider.IsCreationAllowed || !provider.IsAutoCreation {
+	// if auto creation is disabled, send the user to the notFoundOption
+	if !provider.IsAutoCreation {
```

Comment:
Linking is already checked, and you get sent to the "NotFound" page, even if you already know that creation is not permitted, shouldn't we just display an error that creation is not possible?

Reply:
I'm still unsure what you mean. Yes the user is not already linked, auto-linking option would already be handled, but I'm here only checking if the auto creation is enabled (which previously also required `IsCreationAllowed`. If this is not the case, I let the notFoundOption handle the case (as is), which will present the user the possible options, which can be (manual) linking and/or creation.
What i got from the discussion yesterday is, that you want to already check this here and send an error to the function. My argument there is, that the function anyway needs to handle those situations. I assume that's already done, but will check

Reply:
now a specific error is displayed in there aren't any options allowed / possible